.buildkite/scripts/make_image.sh

```diff
@@ -33,12 +33,25 @@ case "$BUILD_TYPE" in
         DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_testnet_${VERSION}"
         LATEST_TAG="latest-val-${IMAGE_ARCH}"
         ;;
+    "val_tpm")
+        echo "Doing a testnet validator with TPM image build for ${IMAGE_ARCH}"
+        DOCKER_BUILD_ARGS="--build-arg BUILDER_IMAGE=$BASE_IMAGE --build-arg RUNNER_IMAGE=$BASE_IMAGE --build-arg REBAR_BUILD_TARGET=docker_testval_tpm $DOCKER_BUILD_ARGS"
+        BASE_DOCKER_NAME="validator_tpm"
+        DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_testnet_${VERSION}"
+        LATEST_TAG="$MINER_REGISTRY_NAME:latest-val-tpm-${IMAGE_ARCH}"
+        ;;
     "validator")
         echo "Doing a mainnet validator image build for $IMAGE_ARCH"
         DOCKER_BUILD_ARGS="--build-arg BUILDER_IMAGE=${BASE_IMAGE} --build-arg RUNNER_IMAGE=${BASE_IMAGE} --build-arg REBAR_BUILD_TARGET=docker_val ${DOCKER_BUILD_ARGS}"
         BASE_DOCKER_NAME="validator"
         DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_${VERSION}"
         ;;
+    "validator_tpm")
+        echo "Doing a mainnet validator with TPM image build for $IMAGE_ARCH"
+        DOCKER_BUILD_ARGS="--build-arg BUILDER_IMAGE=${BASE_IMAGE} --build-arg RUNNER_IMAGE=${BASE_IMAGE} --build-arg REBAR_BUILD_TARGET=docker_val_tpm ${DOCKER_BUILD_ARGS}"
+        BASE_DOCKER_NAME="validator_tpm"
+        DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_${VERSION}"
+        ;;
```

Comment:
we can probably remove the validator builds though, right? we don't need them any more?

Reply:
For the history here, @Vagabond and @jadeallenx. we recommended just trying to build a validator version initially just to get some chain follower running on the freedomfi hardware. but the miner_tpm build seems to be the only one needed now.

Reply:
Ah ok. I'm going to pull in this branch to a local fork of miner and beat on it a little bit